<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>800</width>
                <height>600</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                    <number>0</number>
                </property>
                <property name="leftMargin">
                    <number>0</number>
                </property>
                <property name="topMargin">
                    <number>0</number>
                </property>
                <property name="rightMargin">
                    <number>0</number>
                </property>
                <property name="bottomMargin">
                    <number>0</number>
                </property>
                <item>
                    <widget class="QWidget" name="visualizationWidget" native="true">
                        <layout class="QHBoxLayout" name="visualizationLayout">
                            <property name="spacing">
                                <number>0</number>
                            </property>
                            <property name="sizeConstraint">
                                <enum>QLayout::SetNoConstraint</enum>
                            </property>
                            <property name="leftMargin">
                                <number>0</number>
                            </property>
                            <property name="topMargin">
                                <number>0</number>
                            </property>
                            <property name="rightMargin">
                                <number>0</number>
                            </property>
                            <property name="bottomMargin">
                                <number>0</number>
                            </property>
                        </layout>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QDockWidget" name="dockWidgetInfo">
            <property name="minimumSize">
                <size>
                    <width>236</width>
                    <height>107</height>
                </size>
            </property>
            <property name="features">
                <set>QDockWidget::NoDockWidgetFeatures</set>
            </property>
            <property name="windowTitle">
                <string>Information</string>
            </property>
            <attribute name="dockWidgetArea">
                <number>8</number>
            </attribute>
            <widget class="QWidget" name="dockWidgetInfoContents">
                <property name="enabled">
                    <bool>true</bool>
                </property>
                <layout class="QHBoxLayout" name="dockWidgetInfoLayout">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <property name="sizeConstraint">
                        <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <property name="leftMargin">
                        <number>0</number>
                    </property>
                    <property name="topMargin">
                        <number>0</number>
                    </property>
                    <property name="rightMargin">
                        <number>0</number>
                    </property>
                    <property name="bottomMargin">
                        <number>0</number>
                    </property>
                </layout>
            </widget>
        </widget>
        <widget class="QDockWidget" name="dockWidgetModules">
            <property name="minimumSize">
                <size>
                    <width>110</width>
                    <height>138</height>
                </size>
            </property>
            <property name="features">
                <set>QDockWidget::NoDockWidgetFeatures</set>
            </property>
            <property name="windowTitle">
                <string>Modules</string>
            </property>
            <attribute name="dockWidgetArea">
                <number>2</number>
            </attribute>
            <widget class="QWidget" name="dockWidgetModulesContents">
                <layout class="QHBoxLayout" name="dockWidgetModulesLayout">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <property name="sizeConstraint">
                        <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <property name="leftMargin">
                        <number>0</number>
                    </property>
                    <property name="topMargin">
                        <number>0</number>
                    </property>
                    <property name="rightMargin">
                        <number>0</number>
                    </property>
                    <property name="bottomMargin">
                        <number>0</number>
                    </property>
                </layout>
            </widget>
        </widget>
        <widget class="QDockWidget" name="dockWidgetRobots">
            <property name="features">
                <set>QDockWidget::NoDockWidgetFeatures</set>
            </property>
            <property name="windowTitle">
                <string>Robots</string>
            </property>
            <attribute name="dockWidgetArea">
                <number>1</number>
            </attribute>
            <widget class="QWidget" name="dockWidgetRobotsContents">
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <property name="sizeConstraint">
                        <enum>QLayout::SetNoConstraint</enum>
                    </property>
                    <property name="leftMargin">
                        <number>0</number>
                    </property>
                    <property name="topMargin">
                        <number>0</number>
                    </property>
                    <property name="rightMargin">
                        <number>0</number>
                    </property>
                    <property name="bottomMargin">
                        <number>0</number>
                    </property>
                </layout>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
